src/contexts/ThemeContext.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
To keep the code consistent let's avoid this type of import and only get what we need.